### PR TITLE
planner: consider agg func type in cost model

### PR DESCRIPTION
**Pull request by lzmhhh123**
### What problem does this PR solve? 
After PR #11926. We found that aggregation didn't push to cop task for TPCH Q1. The reason is explained by issue #11948. This PR changes the cost model to consider agg function types simply.

### What is changed and how it works?
Add factor for each agg function instead of the number of agg functions.

### Check List 

Tests 

 - Unit test
 - Integration test

Side effects

 - Possible performance regression
 - Increased code complexity

### CODE REVIEW

File: planner/core/find_best_task.go [hunk 1]
```diff
@@ -43,6 +44,24 @@ const (
 	distinctFactor  = 0.8
 )
 
+var aggFuncFactor = map[string]float64{
+	ast.AggFuncCount:       1.0,
```
Comment (winoros) [inline planner/core/find_best_task.go]: How do you dicide their values?
And what is `defalut` for?

File: planner/core/task.go [hunk 2]
```diff
@@ -873,15 +873,22 @@ func (p *PhysicalStreamAgg) attach2Task(tasks ...task) task {
 
 // GetCost computes cost of stream aggregation considering CPU/memory.
 func (p *PhysicalStreamAgg) GetCost(inputRows float64, isRoot bool) float64 {
-	numAggFunc := len(p.AggFuncs)
-	if numAggFunc == 0 {
-		numAggFunc = 1
+	aggFactorSum := 0.0
+	for _, agg := range p.AggFuncs {
```
Comment (lamxTyler) [inline planner/core/task.go]: Why not extract a function for them?

File: planner/core/task.go [hunk 3]
```diff
@@ -873,15 +873,22 @@ func (p *PhysicalStreamAgg) attach2Task(tasks ...task) task {
...
+		if fac, ok := aggFuncFactor[agg.Name]; ok {
+			aggFactorSum += fac
+		} else {
+			aggFactorSum += aggFuncFactor["default"]
+		}
+	}
+	if aggFactorSum == 0 {
+		aggFactorSum = 1.0
```
Comment (lamxTyler) [inline planner/core/task.go]: So no agg func has more cost that single `bitXXX`? That seems weird.

File: planner/core/find_best_task.go (hunk 1 above)
Comment (lzmhhh123) [reply planner/core/find_best_task.go]: 1. The values are how they keep the result of their arguments. For example:
```
avg: avg.sum += expr, avg.cnt += 1
var_pop: var_pop.sum_sqr += expr*expr, var_pop.cnt += 1, var_pop.sum += expr
```

2. The `default` is just for safety.

File: planner/core/task.go (hunk 3 above)
Comment (lzmhhh123) [reply planner/core/task.go]: My opinion is that bit operation costs lower than other base operations.

**Pull request merged**